doc/rtd/howto/ubuntu_test_prerelease.rst

```diff
@@ -0,0 +1,67 @@
+.. _ubuntu_test_pre_release:
+
+Test pre-release cloud-init
+===========================
+
+After the cloud-init team creates an upstream release, cloud-init will
+be released in the -proposed APT repository for a
+:ref:`period of testing<sru_testing>`. Users are encouraged to test their
+workloads on this pending release so that bugs can be caught and fixed prior
+to becoming more broadly available via the -updates repository. This guide
+describes how to test the pre-release package on Ubuntu.
+
+Add the -proposed repository
+----------------------------
+
+This repository will contain the cloud-init package shortly after the upstream
+release.
+
+.. code-block:: bash
+
+    echo "deb http://archive.ubuntu.com/ubuntu $(lsb_release -sc)-proposed main" >> /etc/apt/sources.list.d/proposed.list
+    apt update
+
+Install the pre-release cloud-init package
+------------------------------------------
+
+.. code-block:: bash
+
+    apt install cloud-init
+
+Test the package
+----------------
+
+Whatever workload you use cloud-init for in production is the best one
+to test. This ensures that you can discover and report any bugs that the
+cloud-init developers missed during testing before cloud-init gets
+released more broadly.
+
+.. code-block:: bash
+
+   # you know what to do
```

Comment:
What is the point of this code block? If it's to say "insert your own test procedure here", that's not super obvious and I don't think we should keep it.